app/views/users/_following_modal.html.erb

```diff
@@ -18,7 +18,7 @@
               <% end %>
             </ul>
           <% else %>
-            <% if @user.id == current_user.id %> 
+            <% if current_user and @user.id == current_user.id %>
```

Comment:
1) users/_following_modal.html.erb when not logged in should indicate the user is not following anyone
     Failure/Error: expect(page).to have_text "This user is not following anyone."
       expected to find text "This user is not following anyone." in "Following: 0 You are not following anyone. Close"
     # ./spec/views/users/_following_modal.html.erb_spec.rb:66:in `block (3 levels) in <top (required)>'